src/main/java/com/pelensky/contactmanager/CLInterface.java

```diff
@@ -0,0 +1,80 @@
+package com.pelensky.contactmanager;
+
+import java.io.*;
+import java.util.Scanner;
+
+
+class CLInterface {
+
+    Scanner input;
+    PrintStream output;
+    private String selectedOption;
+    private Boolean appRunning = true;
+    private ContactList contactList;
+
+
+
+    CLInterface(Scanner input, PrintStream output, ContactList contactList) {
+        this.input = input;
+        this.output = output;
+        this.contactList = contactList;
+    }
+
+    void runApp() {
+        printToConsole("Contact Manager");
+        while (appRunning) {
+            printToConsole("Type `new` to add a new contact\nType `quit` to quit.");
+            makeSelection();
+        }
+    }
+
+
+    private void printUsersSelection() {
+        printToConsole(selectedOption);
+    }
+
+    private void makeSelection() {
+        String selection = input.nextLine().toLowerCase();
+        switch (selection) {
```

Comment:
There is a more OO approach to this - generally try to avoid switch and prefer objects instead. I think this is touched upon in Implementation Patterns.